main.tf

```diff
@@ -911,7 +911,7 @@ resource "aws_flow_log" "agentless_scan_vpc_flow_log" {
 
   # Send logs to manged S3 bucket.
   log_destination_type = "s3"
-  log_destination      = "arn:aws:s3:::${local.prefix}-bucket-${local.suffix}/sidekick/flow-logs/"
+  log_destination     = aws_s3_bucket.agentless_scan_bucket[0].arn
```

Comment:
probably still want to publish this to the `sidekick/flow-logs` path within the bucket for organization purposes.

Reply:
good catch dude — addressed in a7af82d